### PR TITLE
Styled some forms

### PR DESCRIPTION
**Pull request by alejandroereyes**
- styled forms for new meeting, new room, and company branding

### CODE REVIEW

**Comment by mlgiardina** [comment]
:+1:

File: app/views/companies/_form.html.erb [hunk 1]
```diff
@@ -1,37 +1,42 @@
...
+          <div class="field">
+          <%= f.label :logo %><br>
+          <%= f.text_field :logo %>
+        </div>
 
-  <div class="field">
-    <%= f.label :primary_color %><br>
-    <%= f.text_field :primary_color %>
-  </div>
+        <div class="field">
```
Comment (JMD1986) [inline app/views/companies/_form.html.erb]: what up with this unneccessary whitespace?

File: app/views/meetings/new.html.erb [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
   <div class="row">
+    <div class="col-md-4 col-md-offset-4 form-box">
 
-    <h2>New Meeting</h2>
+      <h2>New Meeting</h2>
 
-
-    <%= render 'form' %>
+      <%= render 'form' %>
+    </div>
```
Comment (JMD1986) [inline app/views/meetings/new.html.erb]: is this div neccessary

**Comment by wrpaape** [comment]
:flames:

**Comment by JMD1986** [comment]
:+1:

**Pull request merged**